specifyweb/frontend/js_src/lib/components/Security/AdminStatusPlugin.tsx

```diff
@@ -28,7 +31,17 @@ export function AdminStatusPlugin({
   const loading = React.useContext(LoadingContext);
   const [user] = useResource(resource);
   const isCurrentUser = userInformation.id === user.id;
-
+  const [allCollections] = useAsyncState(
+    React.useCallback(
+      async () =>
+        fetchCollection('Collection', { limit: 0, domainFilter: false }).then(
```

Comment:
no need to fetch collections again as you can just pass it down as a prop from the parent component - https://github.com/specify/specify7/blob/aa434e5d24f9b88b3a2c2d892d8619f5d35e815e/specifyweb/frontend/js_src/lib/components/Security/User.tsx#L164